packages/analytics-browser/src/config.ts

```diff
@@ -121,15 +121,13 @@ export class BrowserConfig extends Config implements IBrowserConfig {
     this.diagnosticsClient = diagnosticsClient;
 
     // Backward compatibility for fetchRemoteConfig
-    if (fetchRemoteConfig) {
-      if (!this.remoteConfig) {
-        this.remoteConfig = {
-          fetchRemoteConfig: fetchRemoteConfig,
-        };
-      }
-      if (!this.remoteConfig.fetchRemoteConfig) {
-        this.remoteConfig.fetchRemoteConfig = fetchRemoteConfig;
-      }
+    this.remoteConfig = this.remoteConfig || {};
```

Comment:
maybe you want this? otherwise you lose remoteConfig.serverUrl
```suggestion
    this.remoteConfig = remoteConfig || {};
```